framework/packages/abstract-testing/src/lib.rs

```diff
@@ -3,16 +3,138 @@ pub mod map_tester;
 pub mod mock_ans;
 pub(crate) mod mock_querier;
 
+use abstract_std::account::{ConfigResponse as AccountConfigResponse, QueryMsg as AccountQueryMsg};
+use abstract_std::objects::ABSTRACT_ACCOUNT_ID;
+use abstract_std::{
+    manager::state::ACCOUNT_MODULES,
+    objects::{
+        account::TEST_ACCOUNT_ID,
+        module::{ModuleInfo, ModuleVersion},
+        module_reference::ModuleReference,
+        ownership,
+    },
+    proxy::state::ACCOUNT_ID,
+    version_control::state::{ACCOUNT_ADDRESSES, REGISTERED_MODULES},
+    ACCOUNT,
+};
+use cosmwasm_std::testing::MOCK_CONTRACT_ADDR;
 use cosmwasm_std::{
+    from_json,
     testing::{MockApi, MockQuerier, MockStorage},
-    OwnedDeps,
+    to_json_binary, Addr, Binary, Empty, OwnedDeps,
 };
 pub use mock_ans::MockAnsHost;
 pub use mock_querier::{
-    map_key, mock_querier, raw_map_key, wrap_querier, MockQuerierBuilder, MockQuerierOwnership,
+    map_key, raw_map_key, wrap_querier, MockQuerierBuilder, MockQuerierOwnership,
 };
+use module::{TEST_MODULE_ID, TEST_MODULE_RESPONSE};
+use prelude::{AbstractMockAddrs, AbstractMockQuerierBuilder};
 pub type MockDeps = OwnedDeps<MockStorage, MockApi, MockQuerier>;
 
+pub fn mock_querier_builder(mock_api: MockApi) -> MockQuerierBuilder {
+    let raw_handler = move |contract: &Addr, key: &Binary| {
+        // TODO: should we do something with the key?
+        let str_key = std::str::from_utf8(key.as_slice()).unwrap();
+        let abstr = AbstractMockAddrs::new(mock_api);
+
+        if contract == abstr.account.addr() {
+            // Return the default value
+            Ok(Binary::default())
+        } else if contract == abstr.version_control {
+            // Default value
+            Ok(Binary::default())
+        } else {
+            Err(format!(
+                "attempt to query {} with key {}",
+                contract, str_key
+            ))
+        }
+    };
+    let abstr = AbstractMockAddrs::new(mock_api);
+    // panic!("{}", abstr.account.addr().to_string());
+    // cosmwasm1s0t3nh7snq5xz2nfkgqdd4wzcnm8d2x7ercmqf9v4rw8du96ud2qe23ef4
```

Comment:
```suggestion
```
It's true only when `cosmwasm` is prefix in `MockApi`

Reply:
this was left from debugging. :)